phobos_tests_extractor.d

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env dub
+/+ dub.sdl:
+name "check_phobos"
+dependency "libdparse" version="~>0.7.0-beta.2"
++/
+/*
+ * Parses all public unittests that are visible on dlang.org
+ * (= annotated with three slashes)
+ *
+ * Copyright (C) 2016 by D Language Foundation
+ *
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE_1_0.txt or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+*/
+// Written in the D programming language.
+
+import std.stdio;
+import std.conv;
+import std.string;
+import std.file;
+import std.regex;
+import std.range;
+import std.exception;
+import std.algorithm;
+import std.path;
```

Comment:
> On thing I'd like to add to our usual style guidelines is imports should be ordered alphabetically 

Good idea -> https://github.com/Hackerpilot/Dscanner/pull/382

> Also, I wonder how our opinionated community would feel about preferring comma-separated imports instead of one per line.

I prefer line-separated imports because it's easy to add/remove them and the git log is clean.
For such scripts one wants to do sth. like `import std.*` - I have seen quite many people having their convenience scripting package that public imports most of Phobos